### DISCUSSION TITLE
How to copy content of :messages to system clipboard

### DISCUSSION THREAD

**Discussion by AckslD** [neovide/neovide#1527] [Q&A]
If I'm in a normal buffer I can copy to system clipboard with `"+y` or a keybind. But how can I copy something printed from `:messages`? In the terminal I can select and `<C-S-c>` but this does not work in neovide if my understanding is correct?

**Comment by s-cerevisiae** [comment]
This can be achieved with Neovim functionalities.
First there is an `execute` function, which captures the output of any command as a string.
To copy the content to clipboard, you can use `:let @+ = <content>`, where `+` is the register name of system clipboard as in `"+`.
So `:let @+ = execute("messages")` will do the job for you.

**Reply by AckslD** [reply]
Thanks for you reply @s-cerevisiae, in the end to make this a bit easier I made a [plugin](https://github.com/AckslD/messages.nvim) for this :)
